### DISCUSSION TITLE
Cabin Ready - Doors  Open

### DISCUSSION THREAD

**Discussion by RnR62** [flybywiresim/aircraft#4821] [Q&A]
I recently committed the crime of taking off with a cabin door open despite getting the verbal and displayed "cabin ready". 

My fault obviously for not manually checking but is there likely to be a stage where it won't be possible to get the cabin ready signal if the cabin is not actually secure?  

Thanks.

**Comment by ExampleWasTaken** [comment]
> 
> 
> I recently committed the crime of taking off with a cabin door open despite getting the verbal and displayed "cabin ready".
> 
> My fault obviously for not manually checking but is there likely to be a stage where it won't be possible to get the cabin ready signal if the cabin is not actually secure?
> 
> Thanks.

There are some requirements that need to be met in order for the cabin to become "ready" aka the ECAM memo going from blue to green.

**Reply by RnR62** [reply]
Yes thanks but I was meaning where the TO button check wouldn't work and we don't get the verbal OK from the cabin crew.    In my case the TO button check worked because I got the OK from the cabin crew despite a cabin door being wide open.   It's doesn't just depend on the colour of a text message.

**Reply by ExampleWasTaken** [reply]
I see. I don't know to what extend that is possible right now, but you can file a [feature request](https://github.com/flybywiresim/a32nx/issues/new?assignees=&labels=Enhancement&template=feature_request_beta.yaml) for that if you want.
